src/config/dynamic-theme-fixes.config

```diff
@@ -587,6 +587,13 @@ INVERT
 
 ================================
 
+easypost.com
+
+INVERT
+img
```

Comment:
Please specify a more specific selector. For example if element or its parent has "class" attribute, you can use class selector, like `.class-value`. If inverted images have some specific path, you can use an attribute selector like `img[src="path.jpg"]` or `img[src*="common-pattern"]` if multiple images have some common name part.